eldritch/encounters.py

```diff
@@ -136,10 +136,27 @@ def Administration7(char):
   gain = events.Gain(char, {"dollars": 8})
   arrest = events.Arrested(char)
   return events.PassFail(char, check, gain, arrest)
+def Asylum1(char):
+  check = events.Check(char, "lore", 0)
+  loss = events.Loss(char, {"sanity": 1})
+  gain1 = events.Gain(char, {"clues": 1})
```

Comment:
I recommend making this a single GainOrLoss event (which supports doing gains and losses at the same time). This makes the execution of this event atomic, which matters if the sanity loss drives the player insane.
(If you have a sequence of loss and then gain, but the loss drives the player insane, then the gain won't be executed.)

Reply:
Done